GTG/gtk/browser/main_window.py

```diff
@@ -297,16 +297,32 @@ def _init_about_dialog(self):
         """
         Show the about dialog
         """
+        # These lines should be in info.py, but due to their dynamic nature
+        # there'd be no way to show them translated in Gtk's About dialog:
+
+        translated_copyright = _("Copyright © 2008-%d the GTG contributors.") % datetime.date.today().year
+
+        UNITED_AUTHORS_OF_GTGETTON = [_("many contributors around the world."), # GTK prefixes this intro line with "Created by "
```

Comment:
.. and something else in other languages, which may make this hard to translate properly as done here now.
This should be changed to not rely on whatever GTG prefixes it with. You can't rely on a cut sentenced into multiple strings to work out for all languages and should not do this.

Reply:
Hmm, wouldn't the comment let translators know that they have to see what GTK prefixes there to make a sentence that makes sense, i.e. for example the equivalent of "Authors   GTG has been created by many contributors around the world." ?

Reply:
I may have to change the grammar of the GTK word, or put the GTK word somewhere else than the start of sentence. It just won't work, don't try to make it work.

https://wiki.gnome.org/TranslationProject/DevGuidelines/Never%20split%20sentences

Reply:
OK made it into a full sentence and committed the changes.